### PR TITLE
Plugin API feature: added getInfo method to context.app

### LINKED ISSUE #3974: Add getInfo to context.app on the Plugin API
**Is your feature request related to a problem? Please describe.**
There is no way to get information about the Insomnia application, like the version.

**Describe the solution you'd like**
Create a getInfo method for context.app that returns an object with app info.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by monrax**
- This closes #3974, Closes INS-919
- The method returns an object with the app `version`, `ID`, `name` and `platform` using the helper methods from `common/constants`

### CODE REVIEW

File: packages/insomnia-app/app/plugins/context/app.tsx [hunk 1]
```diff
@@ -95,6 +96,15 @@ export function init(renderPurpose: RenderPurpose = RENDER_PURPOSE_GENERAL): {
...
       },
 
+      getInfo() {
+        return {
+          version: getAppVersion(),
+          appId: getAppId(), 
+          name: getAppName(),
+          platform: getAppPlatform()
+        }
+      },
```
Comment (develohpanda) [inline packages/insomnia-app/app/plugins/context/app.tsx]: As discussed in Slack, let's change this to only return the version and platform for now, and we can add any extra fields as they become necessary in future!

**Review by develohpanda — CHANGES_REQUESTED** [review]
Thanks for the PR! One small update, please 🙌🏽 

Looks like there are a couple of lint errors too, they can be auto-fixed by running `npm run lint:fix` from the root of the repository

File: packages/insomnia-app/app/plugins/context/app.tsx (hunk 1 above)
Comment (develohpanda) [reply packages/insomnia-app/app/plugins/context/app.tsx]: Updated in d84322d 😊

**Review by develohpanda — APPROVED** [review]
🚀 tested with the following example plugin

```js
module.exports.workspaceActions = [
  {
    action: async (context) => {
      const info = context.app.getInfo();
      await context.app.alert('Info', JSON.stringify(info));
    },
    label: 'Show info',
  },
]
```

File: packages/insomnia-app/app/plugins/context/app.tsx (hunk 1 above)
Comment (monrax) [reply packages/insomnia-app/app/plugins/context/app.tsx]: I'm so ashamed I left this unfinished like that. Thank you once again @develohpanda you're the best 🙏️ 🚀️

Comment (develohpanda) [reply packages/insomnia-app/app/plugins/context/app.tsx]: Hey no worries!! It was a very small change, and I'd like to see this get through 😊

**Review by develohpanda — APPROVED** [review]
Strange how there were some additional lint changes that went in; how did you run lint fix? Was it `npm run lint:fix` from the root directory of this project?

Anyway that's okay, I reverted those. Thank you for adding tests! LGTM

**Comment by monrax** [comment]
Yes, I ran it like that but I also thought it was strange how it changed functions into arrow functions and just not accepting them otherwise. Maybe there's something wrong with my setup. Sorry about that, and glad I could help!

**Comment by develohpanda** [comment]
That's all good! I'll have a look at it 😊

**Review by dimitropoulos — APPROVED** [review]
I think this is ready to go!

I noticed there is a z-index problem, but that isn't related to what this PR implements:

https://user-images.githubusercontent.com/15232461/133501061-c60b5401-4bb8-4d4e-9968-50015d0c0194.mp4

**Pull request merged**